retraktarr/retraktarr.py

```diff
@@ -120,21 +109,28 @@ def main():
     )
     parser.add_argument(
         "--config",
-        type=str,
-        help="Specifies configuration file",
+        action="store",
+        nargs="?",
+        const=True,
+        default=None,
+        help="If a path is provided, retraktarr will use this config file, otherwise it outputs default config location.",
     )
     args = parser.parse_args()
 
     if args.version:
         print(f"reTraktarr v{VERSION}")
         sys.exit(0)
-    if args.config:
+
+    if args.config is not True and args.config is not None:
```

Comment:
Why are you checking `if args.config is not True` - why would it even have bool value?

Reply:
This is for when it is either not provided as an option (it would be None) or if it is provided as an option without a parameter, to display the location of the default config file that would be used.

I tried to keep it to action="store" instead of writing a function for the type, which would be the alternative I'm aware of...